Yoast/yoast-components
# Update to styled-components v4

## Summary

This PR can be summarized in the following changelog entry:

* Updated the `styled-components` package to v4+. 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure you have the console open at all times.
* Walk through the examples in the stand-alone app. Make sure no errors or warnings regarding styled components occur.
* Link `yoast-components` to `wordpress-seo` and check out `snippet-preview-modal-background-color`
* In `wordpress-seo` install all dependencies and build the project.
* Walk through all react components in the plugin.
  * Onboarding wizard
  * Dashboard widget
  * Metabox
  * Sidebar (gutenberg)
  * Search appearance pages

## UI changes

Fixes #
